.github/workflows/release_snapshot.yml

```
@@ -52,69 +51,47 @@ jobs:
            --operation=create
           source ./result.env
           echo "$CREATED_TAG" 
-          echo "created_tag=$CREATED_TAG" >> $GITHUB_OUTPUT
-  linux:
-    needs: linux-create
-    runs-on: ubuntu-latest
+          echo "created_tag=$CREATED_TAG" >> "$GITHUB_OUTPUT"
+          echo "created_tag=$CREATED_TAG" >> $GITHUB_ENV
+      - uses: ./.github/actions/test_install
+        with:
+          version_number: ${{ env.created_tag }}
+          python_version: "3.8"
+          os_platform: "linux"
+  build-for-os-and-python-versions:
```

Add spaces between all the steps + jobs.  It makes it a bit easier to read.

```suggestion
          os_platform: "linux"

  build-for-os-and-python-versions:
```